.github/workflows/sanity.yml

```diff
@@ -183,7 +187,7 @@ jobs:
                   - -c
                   - |
                     cd tests
-                    pytest -m "${{ github.event.inputs.marker_expression || 'sanity' }}" -s -vvv --testbed="${{ matrix.testbed }}" --skip-testrail --alluredir=/tmp/allure-results
+                    pytest -m "${{ github.event.inputs.marker_expression || 'sanity and not interop' }}" -s -vvv --testbed="${{ matrix.testbed }}" --skip-testrail --alluredir=/tmp/allure-results -o --build=${{ github.event.inputs.build_url || "0"}}
```

Comment:
Think the parameter name should be a bit more expressive. `build` can mean a lot of things. Would prefer something like `ap_firmware_url`

Reply:
done